storage/src/mmr/mem.rs

```diff
@@ -26,22 +32,28 @@ impl<H: CHasher> Mmr<H> {
         Self {
             hasher: H::new(),
             nodes: Vec::new(),
+            oldest_remembered_pos: 0,
         }
     }
 
     pub fn size(&self) -> usize {
-        self.nodes.len()
+        self.nodes.len() + self.oldest_remembered_pos as usize
```

Comment:
nit: I think size should just include the number of nodes? 

If we want to add a `latest` function for latest position, I'd be open to that (should return `u64`)

Reply:
size() is a well defined term that is independent of implementation-specific functionality like forgetting (see mod.rs).  I've added a function comment to clarify this, and added a function to obtain the oldest remaining element, which is useful to know what you can still prove, and also for deriving how many nodes are explicitly being maintained in memory.

Reply:
oops fixed